modules/tinymce/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Search and replace plugin would incorrectly find matching text inside non-editable root elements. #TINY-10162
 - Search and replace plugin would incorrectly find matching text inside SVG elements. #TINY-10162
 - Removed use of `async` for editor rendering which caused visual blinking when reloading the editor in-place. #TINY-10249
+- A `list` inside a `li` surrounded by 2 not `li` elements was just considered a `content` of that `li`. #TINY-10268
```

Comment:
```suggestion
- List items containing both text and a nested list caused some list operations to fail. #TINY-10268
```

I'm not sure if "fail" is the correct word, but it's close enough I guess?

Reply:
@TheSpyder since we are managing a nested list between 2 text, could it be better something like this?
```
- List items containing a nested list between two text caused some list operations to fail. #TINY-10268
```

Reply:
Well that was my point, the bug isn't just for between two text items. But if that's all you're going to address in this ticket that seems fine - just seems like a waste to make it a whole new ticket when the fix looks like only one or two lines of code different to what's in this PR.

Reply:
changed based to this comment: https://github.com/tinymce/tinymce/pull/9086#discussion_r1367003088